src/mono/mono/mini/aot-compiler.c

```diff
@@ -14355,6 +14382,50 @@ add_interp_in_wrapper_for_sig (MonoAotCompile *acfg, MonoMethodSignature *sig)
 	add_method (acfg, wrapper);
 }
 
+static gboolean
+mono_aot_add_direct_pinvoke (MonoAotCompile *acfg, char *dpi)
+{
+	char *direct_pinvoke = g_strdup (dpi);
+	if (!direct_pinvoke)
+		return TRUE;
+
+	if (direct_pinvoke[0] == '#')
+		return TRUE;
+
+	g_strstrip (direct_pinvoke);
```

Comment:
If you are going to check direct_pinvoke you need to assign it the return of g_strstrip, but it will always be the same as direct_pinvoke or NULL, so maybe just do a check, if (!g_strstrip(direct_pinvoke))

Reply:
Thanks, changed to the suggested check. `direct_pinvoke` would be modified in place so it wouldnt need to be assigned to the return would it?

Reply:
Yes, the pointer will be stable, and memory will be changed in place.